### ISSUE TITLE
Robot bringup

### ISSUE THREAD

**Issue by jonahsnider** [team581/2024-offseason-bot#20]
Bringup for just base robot functionality.

Blocked by:
- #1
- #3
- #4 
- #5 
- #19
- #17
- #18
- #23 
- #24
- #35
- #36
- #39

**Comment by fcuellar13** [comment]
- Motor ID spreadsheet: 
https://docs.google.com/spreadsheets/d/1zQ2vGFRMg89XwaUJMt96G3RbRwWmUXK5Ax2poAKG11Y/edit?gid=0#gid=0

**Comment by jonahsnider** [comment]
can you make a new issue please. to fill out the spreadsheet + update the motor/sensor IDs in code also @fcuellar13

**Comment by jonahsnider** [comment]
and also follow this structure/format https://docs.google.com/spreadsheets/d/1wlHNQZMmbUz3Kliu5ufz-aywf32sHyXjAL9panAKorU/edit?usp=drivesdk
